app/lib/iiif_manifest_factory.rb

```diff
@@ -1,22 +1,64 @@
 # frozen_string_literal: true
 
 class IiifManifestFactory
-  attr_reader :oid, :manifest, :parent_object
+  attr_reader :oid, :manifest, :parent_object, :mets_doc
 
   require 'iiif/presentation'
 
   def initialize(oid)
     @oid = oid
     @parent_object = ParentObject.find_by(oid: oid)
+    goobi_object = GoobiXmlImport.find_by(oid: oid)
+    return unless goobi_object
```

Comment:
Seems like if we're requesting a iiif manifest, and not finding something that it needs, we'd want to record that as an error condition, yes? I'd be glad to see that captured in a ticket elsewhere so this could be merged now, but I don't want us creating silent failures.

Reply:
I agree, this should throw an exception

Reply:
This should now throw an error, see update.